src/game/Spells/SpellAuras.cpp

```diff
@@ -2214,12 +2214,7 @@ std::pair<unsigned int, float> getShapeshiftModelInfo(ShapeshiftForm form, Unit
                 modelid = 8571;
         }
         else
-        {
-            if (sObjectMgr.GetFactionEntry(target->getFaction())->team == HORDE)
-                modelid = 8571;
-            else
-                modelid = 892;
-        }
+            modelid = 892;
```

Comment:
You sure this won't break anything?

Reply:
It fixes the "Race 0 does not exist in DBC files" error when creatures use a shapeshift spell. It was the previous code that was broken, cause it was trying to use a Player class method on all Units.